scripts/setup.sh

```diff
@@ -0,0 +1,277 @@
+#!/bin/bash
+
+# =============================================================================
+#     CUSTOM FUNCTIONS
+# =============================================================================
+
+error_msg+exit() {
+  echo -e "${1}"
+  exit 1
+}
+
+restart_dspace_container(){
+  echo -en "🔁 Restarting ${1} container...\r"
+  docker container restart "${1}" >> /dev/null
+  if [ $? -ne 0 ]
+  then
+    error_msg+exit "\033[K❌ Error when restarting the ${1} container";
+  fi
+  sleep 10 # TODO :: check if could be removed
+  echo -e "\033[K✅ Restarting ${1} container"
+}
+
+# =============================================================================
+#     MAIN SCRIPT
+# =============================================================================
+
+# Setup script to initialize a new backend docker stack and insert basic data.
+# STEPS:
+#   0. Check script requirement
+#     - Check script argument
+#     - Check required external system command
+#   1. Check that the backend container is running
+#   2. Reinitialize the app
+#     - Clearing the database and migrating it
+#     - Reset data from Solr
+#   3. Create users:
+#     - Admin user to execute future commands
+#     - Create additional users
+#   4. Initialize collections and communities
+#   6. Execute the Python script to make the changes that are not possible via the CLI
+#
+#   X. Register project specific metadata schemas & fields.
+#   X. Assign users into groups
+
+# Define constants used during script execution
+readonly BACKEND="dspace"
```

Comment:
Could be great to be able to run this script for locally installed TOMCAT

Reply:
in next PR ; new feature